### PR TITLE
[zh] sync reference/using-api/api-concepts

### PR DESCRIPTION
**Pull request by jiecloud**
[zh] sync reference/using-api/api-concepts

### CODE REVIEW

**Comment by windsonsea** [comment]
Pls perform the rebase action in your local branch.

**Comment by jiecloud** [comment]
> Pls perform the rebase action in your local branch.

I have completed the rebase operation，Thank you.

File: content/zh-cn/docs/reference/using-api/api-concepts.md [hunk 1]
```diff
@@ -1766,15 +1766,25 @@ Continue Token, Exact
 {{< note >}}
 <!--
 When you **list** resources and receive a collection response, the response includes the
+<<<<<<< HEAD
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/using-api/api-concepts.md:1769]:
```suggestion
<<<<<<< HEAD
```
This tag indicates the rebase is not correct.

**Comment by windsonsea** [comment]
Any of these tags, <<<<, ======, and >>>>>, indicates the rebase is incorrect and incomplete.

File: content/zh-cn/docs/reference/using-api/api-concepts.md [hunk 2]
```diff
@@ -1768,13 +1768,19 @@ Continue Token, Exact
 When you **list** resources and receive a collection response, the response includes the
 [list metadata](/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#listmeta-v1-meta) of the collection as
 well as [object metadata](/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#objectmeta-v1-meta)
+[metadata](/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#listmeta-v1-meta) of the collection as
+well as [object metadata](/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#listmeta-v1-meta)
 for each item in that collection. For individual objects found within a collection response,
 `.metadata.resourceVersion` tracks when that object was last updated, and not how up-to-date
 the object is when served.
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/using-api/api-concepts.md:1775]:
```suggestion
When you **list** resources and receive a collection response, the response includes the
[list metadata](/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#listmeta-v1-meta)
of the collection as well as
[object metadata](/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#objectmeta-v1-meta)
for each item in that collection. For individual objects found within a collection response,
`.metadata.resourceVersion` tracks when that object was last updated, and not how up-to-date
the object is when served.
```

File: content/zh-cn/docs/reference/using-api/api-concepts.md [hunk 3]
```diff
@@ -1768,13 +1768,19 @@ Continue Token, Exact
...
 -->
 当你 **list** 资源并收到集合响应时，
 响应包括集合的[列表元数据](/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#listmeta-v1-meta)。
 以及该集合中每个项目的[对象元数据](/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#objectmeta-v1-meta)。
+响应包括集合的[列表元数据](/zh-cn/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#listmeta-v1-meta)。
+以及该集合中每个项目的[对象元数据](/zh-cn/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#objectmeta-v1-meta)。
+响应包括集合的[元数据](/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#listmeta-v1-meta)
+以及该集合中每个项目的[对象元数据](/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#listmeta-v1-meta)。
 对于在集合响应中找到的单个对象，`.metadata.resourceVersion` 跟踪该对象的最后更新时间，
 而不是对象在服务时的最新程度。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/using-api/api-concepts.md:1785]:
```suggestion
当你 **list** 资源并收到集合响应时，
响应包括集合的[列表元数据](/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#listmeta-v1-meta)
以及该集合中每个项目的[对象元数据](/docs/reference/generated/kubernetes-api/v{{<skew currentVersion >}}/#objectmeta-v1-meta)。
对于在集合响应中找到的单个对象，`.metadata.resourceVersion` 跟踪该对象的最后更新时间，
而不是对象在服务时的最新程度。
```

**Review by windsonsea — COMMENTED** [review]
If you can check the en/zh line by line, it's easy to find the current is still incorrect.

**Comment by windsonsea** [comment]
This will be closed due to long time no response. Feel free to re-open it if you are available, thanks.

/close

**Pull request closed without merging**